lib/loggly/common.js

```diff
@@ -224,7 +224,7 @@ common.loggly = function () {
   //
   // retries to send buffered logs to loggly in every 30 seconds
   //
-  if (timerFunctionForBufferedLogs === null) {
+  if (timerFunctionForBufferedLogs === null && bufferOptions) {
```

Comment:
@mchaudhary @mostlyjason, At this point, if we are using the search functionality of our library like- **(In our app.js file)**

```
client.search('foo', function (err, results) {
  console.dir(results.events);
});
```

Then the library does not set the **bufferOptions** object and when the library tries to access the **bufferOptions.retriesInMilliSeconds** property, an error is thrown so I put this check to enter into this block only when the **bufferOptions** object is set and has some value.